h/templates/activity/search.html.jinja2

```diff
@@ -264,7 +280,7 @@
           </dt>
           <dd class="search-result-sidebar__dd">
             <a href="{{ user.uri }}" class="search-result-sidebar__user-link">
-              {{ user.domain }}
+              {{ pretty_link(user.uri) }}
```

Comment:
Good catch. I thought the `uri` property was converted to an absolute URL before being stored. That isn't the case. If the user enters say "twitter.com/modelstudent" as their link the URL here is `$HOST/users/username/twitter.com/modelstudent`